.gitignore

```diff
@@ -1,3 +1,4 @@
+dist/locale-data
```

Comment:
After looking through everything, I don't think we need to publish the locale-data/ contents to git nor npm.  What do you think?

Reply:
If you agree, I will:

```
git rm -r --cached .
git add .
git commit -m "fixed untracked files"
```

Reply:
Looked at this one more time and believe these aren't needed up in git or npm.  I'll wait for confirmation before removing from this PR though 👍

Reply:
Unlikely they need to be in git, but they should be published to npm

Reply:
@jasonmit Ok.  Removed from .git (that is what you wanted me to do right?)

Reply:
Yup!